sale_isolated_quotation/data/ir_sequence_data.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <!-- Sequences for sale.order -->
+        <record id="seq_sale_quotation" model="ir.sequence">
+            <field name="name">Quotation</field>
+            <field name="code">sale.quotation</field>
+            <field name="prefix">SQ</field>
+            <field name="padding">8</field>
```

Comment:
@lreficent thanks for your point. Actually it should be 3 to inline with the existing sale.order. Fixed.